### ISSUE TITLE
NVDA Alpha 52937 will not uninstall

### ISSUE THREAD

**Issue by SamuelV04** [nvaccess/nvda#19029]
### Brief summary

When I open programs and features in Control Pannel and select NVDA to uninstall, the uninstalling from box is blank and when hitting next the installer says the operation completed successfully, but in reality it took no effect.

### Steps to reproduce

1. Press Windows+R to open the run dialog.
2. Type control. This opens the control pannel.
3. Press P until you reach Programs and Features. Press ENTER.
4. Press N until you reach NVDA, then press enter.
5. Check the edit field to see if there's text in it. This should probably be the NVDA main directory.
6. Press ENTER. You should hopefully see a set of radio buttons to reboot the machine or do it later.

### Actual behavior

No content is in the uninstalling from field. Furthermore, the installer says the action completed successfully, but really nothing happened and no radio buttons were given to restart the computer.

### Expected behavior

The NVDA directory should be in the uninstalling from field, and NVDA should actually uninstall, with the dialog box presenting radio buttons to restart now or restart at a later time.

### NVDA logs, crash dumps and other attachments

I don't think this is aplicable since it is dealing with NVDA's uninstaller and not NVDA itself.

### NVDA type

installed copy

### NVDA version

alpha-52937,f535af1f (2026.1.0.52937)

### Have you tried any other versions of NVDA? If so, please report their behaviors.

NVDA 2025.3 uninstalls fine.

### Windows version

Windows 11, x64, build 10.0.26200.6584

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I do not have any NVDA add-ons installed

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs
